viewer/viewer.js

```diff
@@ -9449,7 +9449,10 @@ app.use(cspHeader, setCookie, (req, res) => {
     huntWarn: Config.get('huntWarn', 100000),
     huntLimit: limit,
     serverNonce: res.locals.nonce,
-    anonymousMode: !!app.locals.noPasswordSecret && !Config.get('regressionTests', false)
+    anonymousMode: !!app.locals.noPasswordSecret && !Config.get('regressionTests', false),
```

Comment:
This context is never used?

Reply:
The anonymous mode part? It's not used in this particular PR, but it is used elsewhere. The diff here is just showing that I added a comma to the end to add more variables.

Reply:
Oh, no just the new business fields. I see you are using them by passing it into this.$constants